selfdrive/manager/manager.py

```diff
@@ -136,6 +134,9 @@ def manager_thread():
     pm_apply_packages('enable')
     start_offroad()
 
+  if spinner:
```

Comment:
Also make sure you do one call to `ensure_running` before you close the spinner to give the ui a chance to start.

Reply:
Thought that would add too much redundant code (sm update, then getting all the arguments correctly) so I put it in the manager loop. Unless it's fine to just do something like `False, False, []`